operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -46,6 +47,20 @@ import (
 const (
 	// NoWorkspacesLabel -> label to be set (to true) when no workspaces are associated to the tenant.
 	NoWorkspacesLabel = "crownlabs.polito.it/no-workspaces"
+	// PVC name
+	UserPvcName = "user-pvc"
+	// NFS secret name
+	NFSSecretName = "user-pvc-secret"
+	// NFS Server key in NFS secret
+	NFSSecretServerNameKey = "server-name"
+	// NFS path key in NFS secret
+	NFSSecretPathKey = "path"
+	// PVC size
+	UserPvcSizeBytes = 1 * 1024 * 1024 * 1024 // 1 Gi
+	// Name of the storage class to use
+	StorageClassName = "rook-nfs"
+	// Namespace of rook-ceph resources
+	RookCephNamespace = "rook-ceph"
```

Comment:
This shall not be hardcoded but parameters.

Reply:
We should get them from the tenant controller's arguments?

Reply:
Yes, exactly

Reply:
Should we do some type of controls or checks for these arguments?

Reply:
Added tenant controller arguments for PVC parameters